### ISSUE TITLE
Regarding retraining figer

### ISSUE THREAD

**Issue by praneethgb** [xiaoling/figer#11]
Hi xiaoling,

I want to re-train Figer with the addition of some data to existing training data but training data available in git hub is serialized using google protobuf. Can you please guide me to modify training data, so that I can add some training data and corresponding label in types.map of it?

**Comment by xiaoling** [comment]
please take a look at #5 #7

**Comment by praneethgb** [comment]
Ok, I will try this and update you if I have any issues.

Thanks

**Comment by praneethgb** [comment]
Hi xiaoling,
 My objective is to add and modify the existing training data attached in https://drive.google.com/file/d/0B52yRXcdpG6MMnRNV3dTdGdYQ2M/view. 
I am able to access the training data using Mention but how do I add and modify the data? what is the format of the training data with features u used to train the perceptron. It would be really helpful if you explain how to add/modify training samples to existing training data.

**Comment by xiaoling** [comment]
(#5) 
> Please take a look at StanfordCoreNLP, which is the tool we use for syntactic analysis. Try using it  on one of the sentences in the training set. Compare the output from StanfordCoreNLP with the ? fields in the protocol buffer file. It would be easy to see how they correspond.
> [Make sure](http://stanfordnlp.github.io/CoreNLP/faq.html#how-can-i-get-original-stanford-dependencies-instead-of-universal-dependencies) that you use the Stanford Dependencies not the new universal ones.

If you prepare the new data in the same way as in the provided training data file, the system will generate features and so on for you. I currently don't have cycles to write the preparation code unfortunately.
